src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/BufferResponsePolicy.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Azure.Base.Http.Pipeline
+{
+    public class BufferResponsePolicy: HttpPipelinePolicy
```

Comment:
nit: If you don't intend instances other than the Singleton to be used, you may want to consider a `private` constructor.

Reply:
We don't do it for other policies yet, so I'll keep it consistent for now.